test/lib/pmd/PmdCatalogWrapper.test.ts

```diff
@@ -61,11 +65,12 @@ describe('PmdCatalogWrapper', () => {
 				before(() => {
 					Sinon.createSandbox();
 					// Spoof a config that claims that only Apex's default PMD JAR is enabled.
-					Sinon.stub(Config.prototype, 'getSupportedLanguages').withArgs(ENGINE.PMD).resolves(['apex']);
+					Sinon.stub(Config.prototype, 'getSupportedLanguages').withArgs(ENGINE.PMD).resolves([LANGUAGE.APEX]);
 					// Spoof a CustomPathManager that claims that a custom JAR exists for plsql, using a weird alias for that language.
 					const customJars: Map<string, Set<string>> = new Map();
-					customJars.set('Pl/SqL', new Set([irrelevantPath]));
+					customJars.set(LANGUAGE.PLSQL, new Set([irrelevantPath]));
```

Comment:
Part of the point of the test is to see if `Pl/SqL` can be properly de-aliased into `plsql`. Using the hardcoded constant undermines that.

Reply:
Whoops, thanks, missed that. Will fix this and below.